tests/fixtures/favicon.html

```diff
@@ -6,9 +6,9 @@
 <link rel="icon" href="favicon.png">
 
 
+
```

Comment:
The script tag was moved to the head so that when the evaluations in the template are blank because there's no script (as is the case for most of the tests), there's one less line in the body and one more line in the head.

I was a little uncertain about removing the option to put the script tag in the body, but perhaps with async available, that's not a problem?